src/libaudtag/id3/id3v24.cc

```diff
@@ -519,6 +521,32 @@ static void add_comment_frame (const char * text, FrameDict & dict)
     g_free (utf16);
 }
 
+static void add_lyrics_frame (const char * text, FrameDict & dict)
```

Comment:
Looks like an exact copy of `add_comment_frame`, except for changing `ID3_COMMENT` to `ID3_LYRICS`. Please combine the two to avoid duplicating code.

Reply:
Yes, I removed the duplicating code from this place (reading). And I did the same thing in the another place (writing). Please check out "id3_associate_memo" and "add_memo_frame" procedures.